### PR TITLE
Configure renovate bot reviewers and auto merge

### PR DESCRIPTION
**Pull request by max-nextcloud**
* Make @mejo- and @max-nextcloud reviewer of renovate PRs
* Set them to automerge

### CODE REVIEW

File: .github/workflows/renovate-approve-merge.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # GitHub actions bot approve
+      - uses: hmarr/auto-approve-action@v3
+        if: startsWith(steps.branchname.outputs.branch, 'renovate/')
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      # Enable GitHub auto merge
+      - name: Auto merge
```
Comment (juliushaertl) [inline .github/workflows/renovate-approve-merge.yml:45]: Difference to the other action (for dependabot) is that it has a limit on only minor version updates https://github.com/nextcloud/text/blob/8a8d84c5c9b27e20aa7e2a97b6d09517019b1563/.github/workflows/dependabot-approve-merge.yml#L39

I would feel ok to have this for all in text due to our test coverage but not sure about making this a general action template then with that.

**Review by juliushaertl — APPROVED** [review]

File: .github/workflows/renovate-approve-merge.yml (hunk 1 above)
Comment (max-nextcloud) [reply .github/workflows/renovate-approve-merge.yml:45]: I think major version bumps are really rare. But I might be wrong. How about we start with limited to minor and then remove the restriction if we they become an issue?

Comment (juliushaertl) [reply .github/workflows/renovate-approve-merge.yml:45]: I think the new actions don't know about the renovate version numbers so we cannot limit unless we find (or write an action) to check that, but fine as it is for now I'd say.

Comment (max-nextcloud) [reply .github/workflows/renovate-approve-merge.yml:45]: Ah.. now i got it. So now we don't have the limit. Fine with me aswell.

**Pull request merged**